composer.json

```diff
@@ -39,7 +39,10 @@
     "require": {
         "php": "^5.5.0|^7.0",
         "payum/iso4217": "~1.0",
-        "guzzlehttp/guzzle": "~6.0",
+        "php-http/httplug": "^1.0",
```

Comment:
any reason to require it explicitly. I personally dont think we need it since we have a `php-http/client-implementation` here. Do I miss something?